src/ert/scheduler/openpbs_driver.py

```diff
@@ -30,7 +29,7 @@
 logger = logging.getLogger(__name__)
 
 _POLL_PERIOD = 2.0  # seconds
-JobState = Literal[
+JOB_STATES = [
```

Comment:
The issue is that we have the same pattern; ie. Literal states in other drivers too.

Reply:
This was only used to iterate over in the openpbs driver and in the test so it didn't need to be a Literal, just a list. The other ones seem to be also used for typing.

Reply:
Seems to be fine though.